objects/Ceiling.js

```diff
@@ -0,0 +1,67 @@
+import {defs, tiny} from '../examples/common.js';
+import CustomObject from './CustomObject.js';
+
+const {
+    Vector, Vector3, vec, vec3, vec4, color, hex_color, Shader, Matrix, Mat4, Light, Shape, Material, Scene,
+} = tiny;
+
+const {Cube, Capped_Cylinder} = defs;
+
+// Seat: class that renders train seats
+export default class Ceiling extends CustomObject {
+    constructor() {
+        super();
+        this.shapes = {
+            cube: new Cube(),
+            cylinder: new Capped_Cylinder(1, 50)
+        };
+
+        this.materials = {
+            ceiling: new Material(new defs.Phong_Shader(),
+                {ambient: .4, diffusivity: .6, color: hex_color("#d1a366")}),
+            lights: new Material(new defs.Phong_Shader(),
+                {ambient: 1, diffusivity: .6, color: color(1, 1, 1, 0.8)}),
+        }
+    }
+
+    /* Custom object functions */
+    render(context, program_state, num_lights=3, ceiling_width=10, ceiling_length=40, ceiling_height=20, model_transform=Mat4.identity(), ) {
+      const ceiling_thickness = 0.25;
+
+      const ceiling_transform = model_transform
+                                  .times(Mat4.translation(0, ceiling_height, 0))
+                                  .times(Mat4.scale(ceiling_length / 2, ceiling_thickness, ceiling_width / 2)); // length & width / 2 bc 2x2x2 cube

```

Comment:
In my code I call the parameter "scales" (i.e. width_scale) since I'm scaling the 2x2x2 cube, as opposed to dividing by 2 to make everything a unit. If we really care about making the code consistent then I can either change my implementation or you can change yours

Reply:
I was thinking that once we start putting the objects together it would be easier to think in terms of the actual units instead of the scaled units. 

When I was creating the train car end walls (#23), it was also easier for me to create when thinking in terms of the actual units instead of the scaled factor since certain lengths were calculated from combinations of other lengths.